contracts/rollup/BatchManager.sol

```diff
@@ -79,25 +78,24 @@ contract BatchManager is Parameters {
         bytes32 depositSubTreeRoot;
         uint256 totalSlashings = 0;
         for (
-            uint256 batchID = batches.length - 1;
+            uint256 batchID = nextBatchID - 1;
             batchID >= invalidBatchMarker;
             batchID--
         ) {
             if (gasleft() <= paramMinGasLeft) break;
 
-            Bitmap.setClaimed(batchID, withdrawalBitmap);
```

Comment:
We can't `setClaimed` here.
- The malicious committer can't claim the stake because the batch is deleted.
- If setClaimed, a future good committer will not be able to claim the batch of this batchID.

Reply:
Good catch